olefile/olefile.py

```diff
@@ -1071,6 +1072,15 @@ def __init__(self, filename=None, raise_defects=DEFECT_FATAL,
             self.open(filename, write_mode=write_mode)
 
 
+    def __enter__(self):
```

Comment:
Just a question: why do you call open() in __enter__, whereas it should have already been called in __init__?

Reply:
That's a good question. I just pushed a new commit removing it, didn't realise that wasn't needed.